template/root-nginx-conf.ejs

```diff
@@ -112,11 +132,22 @@
         location /.well-known/acme-challenge/ {
             root <%-captain.staticWebRoot%>;
         }
-        
+
         # Used by CapRover for health check
         location /.well-known/captain-identifier {
             root <%-captain.staticWebRoot%>;
         }
+
+
+        <%
+        if (captain.logAccessPath) {
+        %>
+        location /goaccess {
+            alias <%-captain.logAccessPath%>;
```

Comment:
Yeah this is what I was talking about in the channel.  Using `express.static` would require the shared volume to be mounted to the `captain-captain` container as well.  Is that what you want?  Right now it's just shared between the nginx and goaccess containers. This would let us dynamically 

Like I said before I tried to get a "proxy" working to have the app serve the files with them only exposed through that top block in the `root-nginx-conf` but wasn't able to get that to work.

Reply:
Hmm... not sure if I am following... 
`captain-captain` should have access to the whole `/captain` directory. I assume these reports are somewhere in the `/captain` directory, no?

Reply:
They're not currently. They're in a new shared docker volume that's only attached to the nginx and goaccess containers.

Reply:
ah I see `nginxLogsVolumeName`....

Hmm... Why can't we use something like `/captain/data/goaccess-logs` ? That way all containers can have access to this.

Reply:
Ahh yeah, didn't know that was a mounted directory shared by the captain instance.  

So I've updated to use that instead of the shared volume and also dynamically update the report when the API is requested.  That required a bit of refactoring the dockerAPI to allow for creating one shot containers but think it turned out well.  

So now all the reports are served through the API and not exposed through the nginx configs at all.

Lemme know what you think

Reply:
Had a quick look and it looks great! I'll be a bit busy in the next week or so, but I'll review this as soon as I get a chance. Looking forward to testing this out! Thanks!